documentation/Using-the-SDK/Advanced/Disposable-Extensions.md

```diff
@@ -1,37 +1,41 @@
 # Abstract
 
-This document outlines making your Custom Data Sources and Extensions
-disposable.
+This document outlines how to make a `Custom Data Source` or SDK
+extension disposable.
```

Comment:
maybe do "extension - such as a `Table` or `DataCooker` - "

Reply:
I'll do Data Cooker as Tables are logical and the SDK has no concept of an instance of a table, rather just a collection of functions. I don't want to suggest that there is a table interface or base class at this time, as that would be confusing.